pydicom/config.py

```diff
@@ -223,3 +223,39 @@ def debug(debug_on=True, default_handler=True):
 
 # force level=WARNING, in case logging default is set differently (issue 103)
 debug(False, False)
+
+
+def empty_value_for_VR(VR, raw=False):
+    """Return the value for an empty element for `VR`.
```

Comment:
Can you add the function to `doc/reference/config.rst`

Reply:
> Can you add the function to `doc/reference/config.rst`

See my review comment below - I would prefer to move this out of config.

Reply:
I'll move it back where it was - I moved it because of circular dependencies, but I can use a local import instead.